config/CMakeLists.txt

```
@@ -121,6 +121,35 @@ configure_file (
   )
 
 # generate version definitions
+if(NVIM_VERSION_MEDIUM)
+  message(STATUS "NVIM_VERSION_MEDIUM: ${NVIM_VERSION_MEDIUM}")
+elseif(${CMAKE_VERSION} VERSION_LESS "3.2.0")
+  message(STATUS "Skipping development version handling with CMake < 3.2.0")
```

awkward wording. maybe this is better:

```suggestion
  message(STATUS "Skipping version-string generation, CMake < 3.2.0")
```